components/builder-worker/src/runner/studio.rs

```diff
@@ -160,52 +179,73 @@ impl<'a> Studio<'a> {
         Ok(exit_status)
     }
 
+    #[cfg(windows)]
     fn studio_command(&self) -> Result<Command> {
-        if self.airlock_enabled {
-            let mut cmd = Command::new("airlock");
-            cmd.uid(studio_uid());
-            cmd.gid(studio_gid());
-            cmd.env_clear();
-            cmd.env("HOME", &*STUDIO_HOME.lock().unwrap()); // Sets `$HOME` for build user
-            cmd.env("USER", STUDIO_USER); // Sets `$USER` for build user
-            cmd.arg("run");
-            cmd.arg("--fs-root");
-            cmd.arg(self.workspace.studio());
-            cmd.arg("--no-rm");
-            if self.network_namespace.is_some() {
-                cmd.arg("--use-userns");
-                cmd.arg(self.network_namespace.as_ref().unwrap().userns());
-                cmd.arg("--use-netns");
-                cmd.arg(self.network_namespace.as_ref().unwrap().netns());
-            }
-            cmd.arg(&*STUDIO_PROGRAM);
+        self.studio_command_not_airlock()
+    }
 
-            Ok(cmd)
+    #[cfg(not(windows))]
+    fn studio_command(&self) -> Result<Command> {
+        if self.airlock_enabled {
+            self.studio_command_airlock()
         } else {
-            let mut cmd = Command::new(&*STUDIO_PROGRAM);
-            cmd.env_clear();
-            debug!("HAB_CACHE_KEY_PATH: {:?}", key_path());
-            cmd.env("NO_ARTIFACT_PATH", "true"); // Disables artifact cache mounting
-            cmd.env("HAB_CACHE_KEY_PATH", key_path()); // Sets key cache to build user's home
-
-            info!("Airlock is not enabled, running uncontained Studio");
-            Ok(cmd)
+            self.studio_command_not_airlock()
         }
     }
+
+    #[cfg(not(windows))]
+    fn studio_command_airlock(&self) -> Result<Command> {
+        assert!(self.airlock_enabled);
+
+        let mut cmd = Command::new("airlock");
+        cmd.uid(studio_uid());
+        cmd.gid(studio_gid());
+        cmd.env_clear();
+        cmd.env("HOME", &*STUDIO_HOME.lock().unwrap()); // Sets `$HOME` for build user
+        cmd.env("USER", STUDIO_USER); // Sets `$USER` for build user
+        cmd.arg("run");
+        cmd.arg("--fs-root");
+        cmd.arg(self.workspace.studio());
+        cmd.arg("--no-rm");
+        if self.network_namespace.is_some() {
+            cmd.arg("--use-userns");
+            cmd.arg(self.network_namespace.as_ref().unwrap().userns());
+            cmd.arg("--use-netns");
+            cmd.arg(self.network_namespace.as_ref().unwrap().netns());
+        }
+        cmd.arg(&*STUDIO_PROGRAM);
+
+        Ok(cmd)
+    }
+
+    fn studio_command_not_airlock(&self) -> Result<Command> {
+        let mut cmd = Command::new(&*STUDIO_PROGRAM);
+        cmd.env_clear();
```

Comment:
I'm assuming the env clear is for some (maybe long forgotten?) reason... @fnichol may be able to shed some light possibly?  Basically I'm just keeping the same model as for Linux builder to keep things as similar as possible.  It may be totally un-necessary for Windows and I can experiment with removing it during the trial of the "-D" flag.